hack/deploy-sbo-local.sh

```diff
@@ -8,6 +8,8 @@ mkdir -p "$OUTPUT"
 
 SBO_LOCAL_LOG="$OUTPUT/sbo-local.log"
 
+export GOCACHE=${GOCACHE:-out/gocache}
```

Comment:
our project does not use gocache at all, all dependencies are vendored, and hence we should build binary first using vendored dependencies, and then start it. We should avoid using `operator-sdk` on line 25 because under the good invokes `go build..`

Reply:
Oh, I didn't realize that.. I'll update the script to use the proper binary, Thx, @pedjak